client/src/schema.graphql

```diff
@@ -0,0 +1,70 @@
+type Query {
```

Comment:
This looks consistent with gitlab graphql types. Has this equivalency check been made?

Reply:
Not sure how to make an equivalency check against the GitLab schema. Their schema is way too big for any online [tools ](https://validator.schema.org/#url=https%3A%2F%2Fgitlab.com%2F-%2Fgraphql-explorer) I could find.

I have manually verified it follows the [reference guide](https://docs.gitlab.com/ee/api/graphql/reference/#pageinfo) and found a few discrepancies, which I have corrected.

Apollo offers some form of [composition checking](https://www.apollographql.com/docs/graphos/delivery/schema-checks/#composition-checks). It would take me a while to figure out, I don't think it is worth it right now.